pkg/controller/kubedirectorcluster/cluster.go

```diff
@@ -619,6 +623,9 @@ func updateStateRollup(
 				// Count missing container as waiting, at this point.
 				if memberStatus.StateDetail.LastKnownContainerState == containerMissing {
 					cr.Status.MemberStateRollup.MembersWaiting = true
+					if memberStatus.StateDetail.SchedulingErrorMessage != nil {
+						cr.Status.MemberStateRollup.MembersNotScheduled = true
```

Comment:
In the block of stuff at the beginning of this function we must also set MembersNotScheduled to false initially.